ui/views_test.py

```diff
@@ -96,6 +97,7 @@ def test_index(client):
     assert response.url == reverse('collection-react-view')
 
 
+@override_settings(ENABLE_VIDEO_PERMISSIONS=False)
```

Comment:
Using `override_settings` and the `settings` fixture at the same time might be confusing. Can we set the setting in the test itself instead?

Reply:
✅